tests/ops/qubit/test_parametric_ops.py

```diff
@@ -3553,7 +3699,11 @@ def test_simplify_to_identity(self, op):
 
         num_wires = op.num_wires if op.num_wires is not qml.operation.AnyWires else 2
 
-        unsimplified_op = op(*([0] * op.num_params), wires=range(num_wires))
+        if op == qml.PCPhase:
```

Comment:
Curious if there is a way to avoid hard-coding `op == qml.PCPhase`? Can the code be general enough to detect this different conditions?

Reply:
I tried, but it made everything more messy. The reason it differs from other parametric ops is because it requires an additional, untrainable parameter (dim)

Reply:
Might be good to get an opinion on this from the core side at some point.